### PR TITLE
Some HTML fixes

### PR DESCRIPTION
**Pull request by tkoolen**
Fixed some issues that my HTML IDE revealed. Note that I'm an HTML novice.

### CODE REVIEW

File: underactuated.html [hunk 1]
```diff
@@ -5151,7 +5151,7 @@
   Again, this algorithm <em>is</em> probabilistically complete.  But after expanding 1000 nodes, the tree is basically a mess of node points all right on top of
   each other:
   <figure>
-    <img width="70%" src="figures/random_tree.svg"></img>
+    <img width="70%" src="figures/random_tree.svg">
```
Comment (RussTedrake) [inline underactuated.html]: this one looks incorrect.  Although I do normally end a single tag with /> for img

**Comment by tkoolen** [comment]
Good catch, fixed.

**Comment by RussTedrake** [comment]
great.  thank you!  (which IDE?)

**Comment by tkoolen** [comment]
WebStorm (sticking to jetbrains products).
https://www.jetbrains.com/webstorm/

**Pull request merged**